assets/_scss/components/_alerts.scss

```diff
@@ -0,0 +1,49 @@
+.usa-alert {
+  background-color: $color-grey-200;
+  margin-top: 1.5em;
+  padding: 1em;
+}
+
+.usa-alert-icon {
+  display: table-cell;
+  padding-right: 1rem;
+}
+
+.usa-alert-body {
+  display: table-cell;
+  vertical-align: top;
+}
+
+.usa-alert-heading {
+  margin: {
+    bottom: .15em;
+    top: 0;
+  }
+}
+
+.usa-alert-text {
+  margin: {
+    bottom: 0;
+    top: 0;
+  }
+}
+
+.usa-alert-success {
+  @extend .usa-alert;
```

Comment:
I actually think it's better to not do this and just include the `.usa-alert` class on each element. I remember a case where I tried to do exactly this for Peace corps, and I had to switch back for some reason (I can't remember the reason right now).